src/lightning/app/components/serve/auto_scaler.py

```diff
@@ -257,7 +257,9 @@ async def consumer(self):
             if batch and (is_batch_ready or is_batch_timeout):
                 self._server_status[server_url] = False
                 # find server with capacity
-                asyncio.create_task(self.send_batch(batch, server_url))
+                # Saving a reference to the result of this function, protects the task disappearing mid-execution
+                # https://textual.textualize.io/blog/2023/02/11/the-heisenbug-lurking-in-your-async-code/
```

Comment:
Let's replace it with the Python docs instead, reducing the risk that the link gets outdated
```suggestion
                # https://docs.python.org/3/library/asyncio-task.html#asyncio.create_task
```

Reply:
Updated :)